packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -684,7 +684,7 @@ export class List extends BaseComponent<IListProps, IListState> implements IList
     let currentSpacer = null;
     let focusedIndex = this._focusedIndex;
     let endIndex = startIndex + renderCount;
-    const shouldVirtualize = this._shouldVirtualize();
+    const shouldVirtualize = this._shouldVirtualize(props);
```

Comment:
in line 692 further down (Can I only comment on lines that were changed in github PRs?) we're still using `this.props`.. I think we should be consistent.. either use `props` passed in or the field not mix

Reply:
@dzearing this comment was still open, what do you think? line 692.. think we should use `props` instead of `this.props`?